docs/en/engines/table-engines/mergetree-family/replication.md

```diff
@@ -129,6 +129,33 @@ Example:
 </macros>
 ```
 
+Allow omitting arguments for the Replicated table engine if defaults are specified in configuration file. For example:
+
+```xml
+<default_replica_path>/clickhouse/tables/{shard}</default_replica_path>
+<default_replica_name>{replica}</default_replica_path>
+```
+
+The two examples below will be equivalent in this case.
```

Comment:
They will not be equivalent. With specified above `default_replica_path`, the second CREATE query will be rewritten to
```
CREATE TABLE table_name (
	x UInt32
) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{shard}', '{replica}') 
ORDER BY x
```
(it does not contain `table_name` in `zookeeper_path` by default)

`default_replica_path` should be like `/clickhouse/tables/{shard}/{database}/{table}`

See also #15348